testing/README.md

```diff
@@ -27,7 +27,11 @@ CLUSTER=kubeflow-testing
 NAMESPACE=kubeflow-test-infra
 ```
 
-After starting `kubectl proxy` you can connect to it at
+After starting `kubectl proxy` you can connect to it at 
+
+```
+http://127.0.0.1:8001/api/v1/proxy/namespaces/kubeflow-test-infra/services/argo-ui/
```

Comment:
Remove the reference to the same URL below?

Reply:
Thanks @foxish for the feedback.